docs/projects/slopi-communication.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Searchable Linkable Open Public Indexed (SLOPI) Communication
+nav_order: 3
+parent: Projects
+---
+
+# Searchable Linkable Open Public Indexed (SLOPI) Communication
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## What is it?
+
+Messages written in the SLOPI (pronounced "sloppy") communication style are:
+
+- Searchable: Messages can be found using Google or other search engines.
+- Linkable: Messages have a permalink.
+- Open: Messages are in the open.
+- Public: Messages are public.
+- Indexed: Messages are indexed by search engines.
+
+This phrasing, especially with regard to messages being searchable and linkable, is inspired by the [discussion][] criteria of [Core Infrastructure Initiative (CII) Best Practices][] program.
+
+Examples of communication tools that support the SLOPI style include:
+
+- mailing lists with public archives
+- public forums
+- public issue trackers
+- Gitter
+- IRC channels with public logs
+- XMPP rooms with public logs
+- public roadmaps
+- public kanban boards
+
+While open source projects should endeavor to communicate in the SLOPI style whenever possible, this style is inappropriate for security, code of conduct violations, and telling co-workers on your floor that you brought in donuts.
+
+The SLOPI style can take some getting used to if you are new to open source culture. It's ok to be a little sloppy. Please don't worry if the message you send isn't perfect. The fact that you sent it in the open is appreciated. Open source is getting friendlier. A thick skin isn't as necessary as it was in the past.
+
+[discussion]: https://github.com/coreinfrastructure/best-practices-badge/commit/65ebe74d7bfdf661502978311200d0c32f7b8be8
+[Core Infrastructure Initiative (CII) Best Practices]: https://bestpractices.coreinfrastructure.org
+
+## Why do we need it?
+
+Open source projects are adopting Slack, forgetting the importance of public and open communication. How can your users and contributors know what you're thinking if the majority of discussion and decision making is done in Slack? Did you forget that you have a mailing list with public archives? Have you considered chat tools that support a SLOPI communication style? Should the discussion happen on the public issue tracker?
```

Comment:
Again here we are targeting slack - this is more appropriate because it's on the project detail page - so maybe say something like "open source projects are adopting tools *like* Slack..."

Reply:
Sure, I can change this to "tools like Slack" instead.